articles/tokens/access_token.md

```diff
@@ -1,49 +1,115 @@
 ---
 title: Auth0 access_token
+sitemap: false
 ---
 
 # Auth0 `access_token`
 
 ## Overview
 
-The Auth0 access token, usually called `access_token` in the code samples, is an opaque string which is returned by calls that invoke the Auth0 authentication process and is only used to call the Auth0 API.  The Auth0 access token does not contain any claims, cannot be decoded, does not conform with any standard and is not signed.
+The Auth0 access token, usually called `access_token` in the code samples, is a [JSON Web Token](/jwt) which can be used to authorize users of your APIs when using our [API Authentication](/api-auth)
+
+It conforms to an industry standard (IETF RFC 7519) and contains three parts: A header, a body and a signature. The header contains the type of token and the hash algorithm used on the contents of the token.  
+
+The body, also called the payload, contains identity claims about a user.  There are some non-mandatory claims with registered names, for things like the issuer of the token, the subject of the token (who the claims are about), and the time of issuance.  Any number of additional claims with other names can be added, though care must be taken to keep the JWT within the browser size limitations for URLs.  
+
+The third part of the JWT is the signature which is used by the recipient of a JWT to validate the integrity of the information conveyed in the JWT.
 
 The [Auth0.js documentation](/libraries/auth0js) shows a sample of how to get the `access_token`.
 
+::: warning-banner
```

Comment:
Again, I think this is misleading.  Focusing too much on the encoding of the token leads to unnecessary confusion in my opinion.